web/src/components/netflow-topology/element-panel.tsx

```diff
@@ -132,13 +141,13 @@ export const ElementPanelContent: React.FC<{
           </Flex>
           <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsNone' }}>
             <FlexItem>
-              <Text id="fromCount">{getMetricValue(fromCount, metricFunction, metricType)}</Text>
+              <Text id="inCount">{getFormattedValue(inCount, metricType, metricFunction)}</Text>
```

Comment:
@memodi @Amoghrd FYI I changed the ids here, `fromCount` => `inCount` and `toCount` => `outCount` , maybe your test cases need an update? I think it's the only DOM structure I changed in this PR